cmd/minishift/cmd/start.go

```diff
@@ -324,7 +324,7 @@ func startHost(libMachineClient *libmachine.Client) *host.Host {
 	isRestart := cmdutil.VMExists(libMachineClient, constants.MachineName)
 	if !isRestart {
 		fmt.Println("-- Minishift VM will be configured with ...")
-		fmt.Println("   Memory:   ", units.HumanSize(float64((machineConfig.Memory/units.KiB)*units.GB)))
+		fmt.Println("   Memory:   ", units.BytesSize(float64(machineConfig.Memory*units.MiB)))
```

Comment:
> 1000cc is 1L

Right, but 5GB != 5GiB and that's what I am seeing

> Using HumanSize is not correct for this

So why not showing what the user specified? This avoids potential confusion.

> However, storing the value as-is might be possible... but, this is only shown when creating the VM. The valus is stored inside the config.json as:

I am not quite sure what you are saying here.

> But the question here is, do we really need to be fine-grained about the VM creation specifications? It seems people in the field do not care much about precision:

Maybe, I probably would not care so much. However, I do react if I specify one thing, but see a different. 

BTW, what about enforcing the unit? I think this would solve some other issues I am seeing. Why make things so vague?